README.rst

```diff
@@ -47,12 +47,13 @@ focus and features:
 - geared for collaboration within trader communities
 - zero interest in adoption by suits; no corporate friendly license, ever.
 - not built for *sale*; built for *people*
+- tools designed, from the ground up, to allow traders to define their edge and execute it
```

Comment:
hmm, *edge* is kinda 80s-90s speak imo.
I wonder if something like:
*tools designed, from the ground up, to allow traders to express themselves*?